django/thunderstore/api/cyberstorm/views/package_listing.py

```diff
@@ -146,6 +146,7 @@ def get_custom_package_listing(
     qs = (
         PackageListing.objects.active()
         .filter_by_community_approval_rule()
+        .public_list()
```

Comment:
Akin to the previous comment, this should be changed so `.active()` isn't explicitly called, but `.public_list()` does it implicitly, so the only change needed would be to replace `PackageListing.objects.active()` with `PackageListing.objects.public_list()`